src/components/molecules/UserInfo/UserInfo.jsx

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
```

Comment:
Looks good for me but if this is picture and text form header, component should accept onClick props too

Reply:
Can u tell me how i can implement this?

Reply:
Just as children prop